lib/senkyoshi.rb

```diff
@@ -11,6 +11,8 @@
 require "nokogiri"
 require "zip"
 
+Zip.write_zip64_support = true
```

Comment:
What if we changed the name of the PR to 'Add Large File Support'? Then I think it better reflects the reason we are changing these things.